src/pages/AppDetails/AppInfo/AppDescription.tsx

```diff
@@ -132,13 +136,23 @@ class AppDescription extends React.Component<AppDescriptionProps, AppDescription
           </Card>
         </GridItem>
         <GridItem span={6}>
-          <Card>
+          <Card style={{ height: '100%' }}>
             <CardBody>
-              <div>
-                <Text component={TextVariants.h2}>Health Overview</Text>
-                <strong>Health</strong>
-              </div>
-              <HealthIndicator id={app.name} health={this.props.health} mode={DisplayMode.LARGE} />
+              <Title headingLevel="h3" size="2xl">
+                {' '}
+                Health Overview{' '}
```

Comment:
This is out of the scope of this PR, but you could may as well change the title of the Workload title?
![](https://user-images.githubusercontent.com/604313/67477065-9b53e680-f659-11e9-849c-a00168183a7f.png)

I see that the overview on the left is in mayus but the one on the right isn't. Do you mind changing that one to Mayus?
See the mockup: https://marvelapp.com/i9h9g7j/screen/59315434
Otherwise, let me know and I'll open a PR.